src/Symfony/Component/Form/Extension/Core/EventListener/ResizeFormListener.php

```diff
@@ -148,14 +149,16 @@ public function onSubmit(FormEvent $event)
             throw new UnexpectedTypeException($data, 'array or (\Traversable and \ArrayAccess)');
         }
 
-        if ($this->deleteEmpty) {
-            $previousData = $event->getForm()->getData();
+        if ($entryFilter = $this->deleteEmpty) {
+            $previousData = $form->getData();
             foreach ($form as $name => $child) {
+                /* @var $child FormInterface */
```

Comment:
ok, will open PR for that. Where can I find information about standards for inline `@var` phpdoc?